plugins/event_notification/providers/http/lib/KDispatchHttpNotificationEngine.class.php

```diff
@@ -88,6 +88,8 @@ public function sendHttpRequest(KalturaHttpNotificationTemplate $httpNotificatio
 		}
 
 		$curlWrapper = new KCurlWrapper();
+       		$curlWrapper->setOpt(CURLOPT_RETURNTRANSFER, 1);
```

Comment:
why this change required for all http notifications?

Reply:
Fixed. Sorry, this file change was relevant for another PR, I removed this change from here